rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -256,6 +261,45 @@ def delete_map_slot(self):
                     print("failed calling slam delete map service")
                     print(trig_resp.message)
 
+    def save_map_image(self):
+        map_name = self.mapName.text()
+
+        if map_name !='':
+            if map_name.find('.'):
+                map_name = string.replace(map_name, '.', '_')
+        else:
+            if not self.default_map_name:
+                map_name = randomTimeString()
+            else:
+                map_name = self.default_map_name
+                
+        if map_name =='':
```

Comment:
Is it possible this statement gets ever executed? It seems like else statement in LOC 270 covers the same case you are checking here and it seems like by the time you reach this line map_name should be set.

Reply:
No it doesn't. It was left in by mistake!